dask/dataframe/tests/test_groupby.py

```diff
@@ -404,6 +404,18 @@ def test_groupby_get_group():
         assert_eq(ddgrouped.a.get_group(2), pdgrouped.a.get_group(2))
 
 
+def test_dataframe_groupby_nlargest():
```

Comment:
Can you parametrize this by one or two levels: `"location"` and `["location", "category"]`. There's been issues around this in the past.

Reply:
I don't quite understand what you mean. Can you elaborate a bit please? What does `this` in `parametrize this` mean? Do you want me to make 1 test for just `location` and a 2nd test for `["location", "category"]`?

Reply:
I guess you meant using `pytest.mark.parametrize`. Doing it now. Sorry, it was unclear before.

Reply:
Yep, exactly. Roughly

```python
pytest.mark.parametrize("level", ["location", ["location", "category"])
def test_dataframe_groupby_nlargest(level):
    ...
```

And then use `level` in the groupby. If it makes the test unnecessarily complex you can split it into two separate tests.

Reply:
Done! A total of 24 tests have been added!

Reply:
Update: There are 36 tests now :)